### ISSUE TITLE
18501619: Reachability should be available as an accessibility shortcut

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#3038]
#### Description

Summary:
On devices without Touch ID (or minimally the iPhone 5), ideally you should be able to enable Reachability as the accessibility shortcut. With this set as the shortcut, triple pressing the home button would shift the UI down, just like double tapping on a Touch ID enable home button does on the iPhone 5S and up.

Steps to Reproduce:
1. Go to Settings app -> General -> Accessibility -> Scroll to bottom -> Accessibility Shortcut
2. Reachability is not currently an option

Expected Results:
1. Go to Settings app -> General -> Accessibility -> Scroll to bottom -> Accessibility Shortcut -> Choose Reachability
2. Triple press the home button to enable Reachability, which would shift the UI down.

Actual Results:
Reachability is not currently available on non Touch ID devices. It would ideally be available on at least the iPhone 5 via the accessibility shortcut.

Version:
iOS 8.0 and up

Configuration:
iPhone 5
## 

Product Version: 5
Created: 2014-09-30T16:45:58.972955
Originated: 2014-09-30T00:00:00
Open Radar Link: http://www.openradar.me/18501619
